example.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/python3
 
 def add(a,b):
+"""Adding a and b"""
   return a + b
 
-def substract(a,b)
-  return a + b
+def substract(a,b):
+  """Subtractiiiing a and b"""
```

Comment:
Can we fix before merging. Other good effort
```suggestion
  """Subtracting a and b"""
```
@svenvanderburg have a look if spelling is correct now.